lambdas/restore-vcs/README.md

```diff
@@ -1,14 +1,40 @@
 # RestoreVcsHandler
 
-What this lambda is for and does: https://govukverify.atlassian.net/browse/PYIC-4602
+---
 
-To run this lambda, create an appropriate payload including userIds with vcs to be restored in the appropriate environment.
+## Description
+
+This lambda mirrors the functionality of [RevokeVcsHandler](lambdas/revoke-vcs/README.md), except for some differences:
+1. The direction of change of VC ownership is opposite
+2. The lambda does not send a failing audit event
+
+---
+
+## Running the lambda
+
+> This lambda should only be called with explicit permission from stakeholders
+
+### Payload
+
+- The payload is the same as for the revoke lambda.
+
+### Command
 
 ```bash
-aws-vault exec <profile> -- aws lambda invoke --function-name restore-vcs-<env> --invocation-type RequestResponse --payload fileb://restoreVcsLambdaPayload.json response.json
+aws-vault exec PROFILE -- \
```

Comment:
Maybe worth providing a way of doing this with [SSO](https://govukverify.atlassian.net/wiki/spaces/PLAT/pages/3554280423/Getting+Ready+for+AWS+SSO) as I think this is the way we're expected to interact with AWS profiles and sessions moving forward?

Reply:
I think you're right though I think I will merge this now, but it's helpful to know this and I may go back and update it